.github/workflows/rust.yml

```diff
@@ -28,35 +29,37 @@ jobs:
           - os: windows-latest
             CSPICE_URL: "https://naif.jpl.nasa.gov/pub/naif/toolkit//C/PC_Windows_VisualC_64bit/packages/cspice.zip"
             SHELL: powershell
+          - download_cspice: false
+            CSPICE_DIR: '${{ github.workspace }}/cspice'
+
     steps:
       - uses: actions/checkout@v2
 
       - name: Download CSPICE
-        if: ${{ matrix.os != 'windows-latest' }}
+        if: ${{ matrix.os != 'windows-latest' && matrix.download_cspice }}
```

Comment:
if might be easier & clearer to follow - if you just made a separate job to test the download feature ?